docs/self-managed/setup/deploy/amazon/aws-ec2.md

```diff
@@ -0,0 +1,352 @@
+    - (optional) a [Network Load Balancer](https://docs.aws.amazon.com/elasticloadbalancing/latest/network/introduction.html) (NLB) is used to expose the gRPC endpoint of the Zeebe Gateway, in case external applications require it.
+    - (optional) a [Bastion Host](https://en.wikipedia.org/wiki/Bastion_host) to allow access to the private EC2 instances since they're not publicly exposed.
+      - Alternatively, one can utilize the [AWS Client VPN](https://docs.aws.amazon.com/vpn/latest/clientvpn-admin/what-is.html) instead to reach the private subnet within the VPC. The setup requires a lot of extra work and certificates, but one can follow the [getting started tutorial by AWS](https://docs.aws.amazon.com/vpn/latest/clientvpn-admin/cvpn-getting-started.html).
+    - a NAT Gateway that allows the private EC2 instances to reach the internet to download and update software packages. This can not be used to access the EC2 instances.
+- [Security Groups](https://docs.aws.amazon.com/vpc/latest/userguide/vpc-security-groups.html) to handle traffic flow to the VMs.
+- an [Internet Gateway](https://docs.aws.amazon.com/vpc/latest/userguide/VPC_Internet_Gateway.html) as glue component between the VPC and the Internet, to allow traffic between those.
+
+Both types of subnets are distributed over three availability zones of a single AWS region, allowing for a highly available setup.
+
+:::note Single Deployment
+Alternatively, one can run the same setup with a single AWS EC2 instance as well but should be aware that in case of a zone failure, their whole setup would be unreachable.
+:::
+
+## Prerequisites
+
+- An AWS account to create any resources within AWS.
+  - On a high level, permissions are required on the **ec2**, **iam**, **elasticloadbalancing**, **kms**, **logs**, and **es** level.
+  - For a more fine-grained view of the permissions, check this [example policy](https://github.com/camunda/camunda-deployment-references/blob/main/aws/ec2/example/policy.json).
+- Terraform (1.7+)
+- Unix based Operating System (OS) with ssh and sftp
+  - Windows may be used with [Cygwin](https://www.cygwin.com/) or [Windows WSL](https://learn.microsoft.com/en-us/windows/wsl/install) but has not been tested
+
+## Considerations
+
+- TODO: reference official limitations if there are any
+- The Optimize importer is not highly available and must only run once within the whole setup.
+
+## Outcome
+
+The outcome is a fully working Camunda orchestration cluster running in a high availability setup using AWS EC2 and utilizing a managed OpenSearch domain.
+The EC2 instances come with an extra disk meant for Camunda to ensure that the content is separated from the operating system.
+
+## Get started
```

Comment:
I think we can remove "Get started," as this h2 encompasses the entire section, and bump up the following H3s to H2s instead.